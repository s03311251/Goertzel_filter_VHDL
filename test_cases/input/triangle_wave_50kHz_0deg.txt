0000
0666
0CCD
1333
1999
2000
2666
2CCC
3332
3999
3FFF
3999
3332
2CCC
2666
2000
1999
1333
0CCD
0666
0000
0666
0CCD
1333
1999
2000
2666
2CCC
3332
3999
3FFF
3999
3332
2CCC
2666
2000
1999
1333
0CCD
0666
0000
0666
0CCD
1333
1999
2000
2666
2CCC
3332
3999
3FFF
3999
3332
2CCC
2666
2000
1999
1333
0CCD
0666
0000
0666
0CCD
1333
1999
1FFF
2666
2CCC
3332
3999
3FFF
3999
3332
2CCC
2666
2000
1999
1333
0CCD
0666
0000
0666
0CCD
1333
1999
2000
2666
2CCC
3332
3999
3FFF
3999
3332
2CCC
2666
2000
1999
1333
0CCD
0666
