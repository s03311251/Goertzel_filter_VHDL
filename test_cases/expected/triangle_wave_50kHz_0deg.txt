0D0B2
